server/aws-lsp-codewhisperer/src/language-server/localProjectContext/localProjectContextServer.ts

```diff
@@ -164,6 +164,11 @@ export const LocalProjectContextServer =
         const updateConfigurationHandler = async (updatedConfig: AmazonQWorkspaceConfig) => {
             logging.log('Updating configuration of local context server')
             try {
+                if (updatedConfig.isDefaultConfig === true) {
+                    logging.log('Skipping init for default config')
```

Comment:
nit: can be a lil more specific
```suggestion
                    logging.log('Skipping local project context initialization for default config')
```